File: Modules/ServerTypes/TrainingManagement/ProgramParticipantRoleType.ts
import { Decorators } from "@serenity-is/corelib";

export enum ProgramParticipantRoleType {
    Trainee = 0,
    Trainer = 1,
    Staff = 2
}
Decorators.registerEnumType(ProgramParticipantRoleType, 'HRMSoftware.TrainingManagement.ProgramParticipantRoleType', 'HumanResource.TrainingManagement.ProgramParticipantRoleType');

File: Modules/HumanResource/TrainingManagement/ProgramParticipant/ProgramParticipantRoleType.cs
using System.ComponentModel;

namespace HRMSoftware.TrainingManagement;
[Serenity.ComponentModel.EnumKey("HumanResource.TrainingManagement.ProgramParticipantRoleType")]
public enum ProgramParticipantRoleType
{
    [Description("Trainee")]
    Trainee = 0,
    [Description("Trainer")]
    Trainer = 1,
    [Description("Staff")]
    Staff = 2,
}